config/webpack/commonWebpackConfig.js

```
@@ -24,45 +24,49 @@ const ignoreWarningsConfig = {
-      }
-    };
-  } else {
-    sassLoader.options = sassLoader.options || {};
-    sassLoader.options.api = 'modern';
+  if (sassLoaderIndex !== -1) {
+    const sassLoader = scssRule.use[sassLoaderIndex];
+    if (typeof sassLoader === 'string') {
+      scssRule.use[sassLoaderIndex] = {
+        loader: sassLoader,
+        options: {
+          api: 'modern'
```

ESLint

[error] Insert `,`

(prettier/prettier)